README.md

```diff
@@ -504,6 +504,9 @@ IEEE Int. Conf. Computer Vision (ICCV), 2023., [code](https://github.com/ZHU-Zhi
 - <a name="Wang23arxiv"></a>Wang, Z., Molloy, T., van Goor, P., Mahony, R.,<br/>
 *[Event Blob Tracking: An Asynchronous Real-Time Algorithm](https://arxiv.org/abs/2307.10593).*,<br/>
 arXiv:2307.10593, 2023. [PDF](https://arxiv.org/pdf/2307.10593.pdf), [Video](https://www.youtube.com/watch?v=1W2DC7BI-uc), [Project page](https://github.com/ziweiWWANG/Event-Blob-Tracking).
+- <a name="Nagaraj23icra"></a>Nagaraj, M., Liyanagedera, C.M., Roy, K.,<br/>
+*[DOTIE - Detecting Objects through Temporal Isolation of Events using a Spiking Architecture](https://ieeexplore.ieee.org/abstract/document/10161164).*,<br/>
+IEEE International Conference on Robotics and Automation (ICRA), 2023., [ICRA 2023 Full paper](https://ieeexplore.ieee.org/abstract/document/10161164), [Arxiv](https://arxiv.org/abs/2210.00975), [CVPR 2023 workshop](https://tub-rip.github.io/eventvision2023/papers/2023CVPRW_Live_Demonstration_Real-time_Event-based_Speed_Detection_using_Spiking_Neural_Networks.pdf).
```

Comment:
IEEE Int. Conf. Robotics and Automation (ICRA)  , like all other entries please.
No comma after the full stop: 2023. [ ... ]
No need to write the same link twice (https://ieeexplore.ieee.org/abstract/document/10161164), so the part "[ICRA 2023 Full paper]" can be removed.

Reply:
thanks!

Reply:
Made the corrections suggested